src/main/java/com/yurjinia/user/service/UserProfileService.java

```diff
@@ -102,4 +121,10 @@ private UserProfileEntity getUserProfileByEmail(String email) {
                 .orElseThrow(() -> new CommonException(ErrorCode.USER_NOT_FOUND, HttpStatus.NOT_FOUND));
     }
 
-}
+    private void updateStringIfNotBlank(String newValue, Consumer<String> updater) {
```

Comment:
you can move this method to separate class - UserProfileHelper, make it as Utils class (put an annotation), make a static method.

update the name of method - updateMetaData(String value, Consumer<String> metaData);